### ISSUE TITLE
link to participants transaction list

### ISSUE THREAD

**Issue by bytesun** [dfinity/nns-dapp#5671]
add a link to open transaction history for SNS swap

**Comment by csepreghy** [comment]
Hey, could you please explain in more detail what feature you would like to see?

**Comment by bytesun** [comment]
> Hey, could you please explain in more detail what feature you would like to see?

yes, I would like to know the detail participate transaction list in SNS launch, for example, there is a link of direct commitment amount to show the exact transactions history, if not, showing the account address, where those participate funds go,  will helpful too.

![image](https://github.com/user-attachments/assets/0288d730-99c7-4dd1-a4cc-b43da9dd141f)
